tools/bin/commands/util/operator_funcs

```diff
@@ -105,7 +163,7 @@ create_operator_image() {
 
     local build_dir=$(prepare_docker_context "$operator_dir")
     pushd $build_dir > /dev/null
-    trap "rm -rf $build_dir" EXIT
+    CLEANUP_DIRS+=($build_dir)
```

Comment:
this works if the function is not evaled via `$()`. Otherwise, CLEANUP_DIRS can't be easily updated.